graphbolt/src/python_binding.cc

```diff
@@ -69,6 +69,7 @@ TORCH_LIBRARY(graphbolt, m) {
   m.def("unique_and_compact", &UniqueAndCompact);
   m.def("isin", &IsIn);
   m.def("index_select", &ops::IndexSelect);
+  m.def("index_select_csc", &ops::IndexSelectCSC);
```

Comment:
why do we need to expose IndexSelectCSC to python? The reason we expose index_select to python is to support feature fetcher. If IndexSelectCSC is designed only for sampling, should we expose the sampling api only?

Reply:
It is fine if you just want to expose it for unit test, if so, add a comment.

Reply:
To copy indices, we pass (indptr, indices), to copy edge probs, we pass (indptr, probs_or_mask).

Reply:
SGTM.